src/preset.ts

```diff
@@ -66,38 +70,62 @@ const nodeModulesThatNeedToBeParsedBecauseTheyExposeES6 = [
   "uuid",
 ];
 
-const include = new RegExp(
-  `[\\\\/]node_modules[\\\\/](${nodeModulesThatNeedToBeParsedBecauseTheyExposeES6.join(
-    "|"
-  )})`
-);
-
-const es6Loader = {
-  test: /\.js$/,
-  use: [
-    {
-      loader: require.resolve("babel-loader"),
-      options: {
-        sourceType: "unambiguous",
-        presets: [ie11Preset],
+const escapeDirectorySeparators = (module: string) => {
```

Comment:
Since adding this function(`escapeDirectorySeparators`) couldn't the `[\\\\/]` part from some of `nodeModulesThatNeedToBeParsedBecauseTheyExposeES6` items just be replaced with `/`?

Reply:
Good point!